lessons/en/basics/date_time.md

```diff
@@ -1,5 +1,5 @@
 %{
-  version: "1.1.1",
+  version: "1.1.2",
```

Comment:
These seem like great changes to include in other languages, let's bump the minor version so it becomes a call out:

```suggestion
  version: "1.2.0",
```